code/persistentstorage/Databases.md

```diff
@@ -0,0 +1,108 @@
+# Databases
+
+This document decribes the following:
+- [The various databases used for each component in the Storj network.](#Storj-components-with-databases)
+- [How to view the contents of the various databases.](#View-the-Contents-of-a-Database)
+- [Steps on how to modify existing databases.](#Steps-to-Modify-Existing-Databases)
+
+
+## Storj Components With Databases
+
+Each component of the Storj network (i.e. satellites, storage nodes, uplinks) has different persistent data stores.
+
+Satellites have three databases:
+1. SatelliteDB, also called the master database.
+2. PointerDB.
+3. Kademlia Routing Table.
```

Comment:
The master db is the interface that contains all the databases that the satellite needs to work. So, pointer db, kademlia routing table are both part of the master database.

The master database specifies which databases are needed, it doesn't specify how they are implemented. So, different satellites will end up with different backends.